Server/Auth/ApiAuthorizationFilter.cs

```diff
@@ -25,15 +27,26 @@ public void OnAuthorization(AuthorizationFilterContext context)
 
             if (context.HttpContext.Request.Headers.TryGetValue("Authorization", out var result))
             {
-                var keyId = result.ToString().Split(":")[0]?.Trim();
-                var apiSecret = result.ToString().Split(":")[1]?.Trim();
+                var tokenType = result.ToString().Split(" ")[0].Trim();
 
-                if (DataService.ValidateApiKey(keyId, apiSecret, context.HttpContext.Request.Path, context.HttpContext.Connection.RemoteIpAddress.ToString()))
+                switch (tokenType)
                 {
-                    var orgID = DataService.GetApiKey(keyId)?.OrganizationID;
-                    context.HttpContext.Request.Headers["OrganizationID"] = orgID;
-                    return;
+                    case "Basic":
+                        var encodedToken = result.ToString().Split(" ")[1].Trim();
+
+                        byte[] data = Convert.FromBase64String(encodedToken);
+                        string decodedString = Encoding.UTF8.GetString(data);
+                        var keyId = decodedString.ToString().Split(":")[0]?.Trim();
+                        var apiSecret = decodedString.ToString().Split(":")[1]?.Trim();
```

Comment:
Should verify that `decodedString.Split(":")` actually contains more that one element, else this will fail

Reply:
👍

Reply:
@stevesobol  Can you do this?

Reply:
Done.